manifests/03-rbac-role-ns-openshift-config-managed.yaml

```diff
@@ -35,3 +35,16 @@ rules:
   - console-public
   verbs:
   - get
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: managed-configmap-reader
```

Comment:
So currently we have:

```bash
oc get role -n openshift-config-managed
NAME               AGE
console-operator   28h
console-public     28h
machine-approver   28h
```
For this reason I'm inclined to name it `console-configmap-reader`.  Since its in `openshift-config-managed` namespace.  That would give us:

```bash
oc get role -n openshift-config-managed
NAME                              AGE
console-configmap-reader          28h
console-operator                  28h
console-public                    28h
machine-approver                  28h
```

Reply:
We did speak about `console-public` being somewhat "different" as its more for other components being able to get the `console-public` configmap so those components can `oc get configmap console-public -n openshift-config-managed --template '{{.data.consoleURL}}'` and use the console url value.